package.json

```diff
@@ -47,9 +47,9 @@
     "expect": "^1.14.0",
     "mocha": "^3.0.0",
     "nyc": "^10.1.2",
-    "webpack": "^2.2.1"
+    "webpack": "^3.0.0"
   },
   "peerDependencies": {
-    "webpack": "^1.12.0 || ^2.1.0-beta.0 || ^2.1.0"
+    "webpack": "^1.0.0 || ^2.0.0 || >= 3.0.0-rc.0 || ^3.0.0"
```

Comment:
Just doing what [webpack-defaults does](https://github.com/webpack-contrib/webpack-defaults/blob/197df50c796c3499e4dd1f8c8eae9d92fbde764b/src/tasks/package.js#L40) - does it need to be in there @d3viant0ne, given the short run of the RC?